ci-operator/config/openshift/cluster-update-keys/openshift-cluster-update-keys-master.yaml

```
@@ -29,6 +33,10 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-upgrade-aws
+- as: generate-configmap
+  commands: bash hack/test-generate-configmap.sh
```

```suggestion
  commands: bash hack/test-generate-configmap.sh && git diff --exit-code
```
We want to ensure that committed configmaps are valid and don't need to be regenerated

---

Oh, I read "test-generate" as "generate configmap for test instance (==ci)". Perhaps its best to rename it to "verify-configmaps".

No need for additional `git diff` then

---

Thank you for the advice, "verify-configmaps" is a better name.

Renamed the test in the commit https://github.com/openshift/release/commit/938a1a2be06cdd186c75ee8e965133ca09428c9f

And renamed the script in the Pull request https://github.com/openshift/cluster-update-keys/pull/39.